CHANGES.rst

```diff
@@ -5,6 +5,7 @@ unreleased
 ------
 
 - [fix] support for rabbitpy 0.27.x
+- [fix] Random port selection ports accept tuples and sets
```

Comment:
- [feature]  Random port selection ports accept tuples and sets. replace string representation [backward incompatible]